CHANGELOG.md

```diff
@@ -7,7 +7,13 @@ All notable changes to this project will be documented in this file.
 
 #### API breaking changes
 
-TBD
+- Removing `MaskDesignable` public methods:
+   - `maskCircle()`, use instead: `view.maskType = String(MaskType.Circle)`
```

Comment:
@lexrus I think `rawValue` is for the actual value it saves. e.g. if the enum inherits from `Int` then rawValue is a number. But `String()` is the presentation of the option. Here is the code I test

```
enum A : Int {
  case A1
}

// A1
String(A.A1)

// 0
A.A1.rawValue
```

If the enum inherits from `String`, I don't see the deference, but if the enum doesn't inherit from any type, `rawValue` is not available.

```
// B1
String(B.B1)

// Value of type `B` has no member `rawValue`
B.B1.rawValue
```

I think I change them in this commit https://github.com/JakeLin/IBAnimatable/pull/110/commits/954a55070ef61c5e0bd2787b4df22218ef57d69c